corehq/apps/userreports/fixtures.py

```diff
@@ -65,6 +69,8 @@ def __call__(self, user, version, last_sync=None):
                 reports_elem.append(self._report_config_to_fixture(report_config, user))
             except UserReportsError:
                 pass
+            except Exception as err:
+                logger.exception('Error generating report fixture: {}'.format(err))
```

Comment:
any reason you chose to make a new logger? I think that's typically only useful if you intend to treat these logging statements as "special" somehow, which I'm not sure makes sense here. 

it doesn't necessarily hurt, but `logging.exception` would be more consistent with how the rest of HQ would handle a one-off like this.